Applications/Utils/MeshEdit/ResetPropertiesInPolygonalRegion.cpp

```diff
@@ -224,6 +224,18 @@ int main (int argc, char* argv[])
 
 	if (int_property_arg.isSet()) {
 		int int_property_val(int_property_arg.getValue());
+
+		// check if PropertyVector exists
+		boost::optional<MeshLib::PropertyVector<int> &> opt_pv(
+			mesh->getProperties().getPropertyVector<int>(property_name)
+		);
+		if (!opt_pv) {
+			opt_pv = mesh->getProperties().createNewPropertyVector<int>(
+				property_name, MeshLib::MeshItemType::Cell, 1);
```

Comment:
The tuple size is 1 by default...
I had to ask myself what is so special about the 1 in the code...